consensus/hotstuff/votecollector/consensus_cluster_vote_collector.go

```diff
@@ -90,7 +90,7 @@ func (c *ConsensusClusterVoteCollector) AddVote(vote *model.Vote) error {
 	}
 
 	// we haven't collected sufficient weight or shares, we have nothing to do further
-	if !c.hasSufficientStake() {
+	if !c.hasSufficientWeight() {
```

Comment:
I guess just a rider? As long as this is intentional, i think it's fine

Reply:
Yes it's intentional. Now that we have turned the branch protection on, if I don't make this change the tests do not pass and the PR is not mergeable